tests/providers/google/cloud/hooks/test_dataplex.py

```diff
@@ -174,3 +187,129 @@ def test_get_lake(self, mock_client):
             timeout=None,
             metadata=(),
         )
+
+    @mock.patch(DATAPLEX_HOOK_CLIENT)
+    def test_create_zone(self, mock_client):
+        self.hook.create_zone(project_id=PROJECT_ID, region=REGION, lake_id=LAKE_ID, zone_id=ZONE_ID, zone={})
+
+        mock_client.return_value.create_zone.assert_called_once_with(
+            request=dict(
+                parent=ZONE_NAME,
+                zone_id=ZONE_ID,
+                zone={},
+            ),
+            retry=DEFAULT,
+            timeout=None,
+            metadata=(),
+        )
+
+    @mock.patch(DATAPLEX_HOOK_CLIENT)
+    def test_delete_zone(self, mock_client):
+        self.hook.delete_zone(project_id=PROJECT_ID, region=REGION, lake_id=LAKE_ID, zone_id=ZONE_ID)
+
+        mock_client.return_value.delete_zone.assert_called_once_with(
+            request=dict(
+                parent=ZONE_PARENT,
+            ),
+            retry=DEFAULT,
+            timeout=None,
+            metadata=(),
+        )
```

Comment:
This test is failing because of a mistype here `request=dict(...)` instead of `name=dict(...)`.

Reply:
changed